drivers/Makefile.dep

```diff
@@ -152,6 +152,12 @@ ifneq (,$(filter fxos8700,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter extend_gpio,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio
+  FEATURES_REQUIRED += periph_gpio_irq
```

Comment:
Is it really necessary to always use the `periph_gpio_irq` module? IMHO, it should be the other way around. That is, the interrupt-related parts of the API should only be activated if the module `periph_gpio_irq` is used by someone.
```
#if MODULE_PERIPH_GPIO_IRQ
...
#endif
```

Reply:
You're probably right. Fixed.